docs/architecture/adr-027-deterministic-protobuf-serialization.md

```diff
@@ -45,24 +65,38 @@ with the following additions:
    must be omitted
 4. `repeated` fields of scalar numeric types must use
    [packed encoding](https://developers.google.com/protocol-buffers/docs/encoding#packed)
```

Comment:
@amaurymartiny @webmaster128 I think this should apply to non-numeric types as well, otherwise repeated non-numeric types could use either packed or unpacked encoding (a source of malleability). Thoughts?

Reply:
"Only repeated fields of primitive numeric types (types which use the varint, 32-bit, or 64-bit wire types) can be declared "packed"." [source](https://developers.google.com/protocol-buffers/docs/encoding#packed)

I actually don't understand why (I can spin up a spec for pack-encoding repeated strings or bytes), but hey, it is what it is.

Reply:
🤷 it is what it is